tests/unit/adapters/osf-adapter-test.js

```diff
@@ -1,12 +1,401 @@
-import { moduleFor, test } from 'ember-qunit';
+import Ember from 'ember';
+import {
+    moduleFor
+} from 'ember-qunit';
+import test from 'dummy/tests/ember-sinon-qunit/test';
+import FactoryGuy, {
+    manualSetup
+} from 'ember-data-factory-guy';
+
+import DS from 'ember-data';
+import OsfAdapter from 'ember-osf/adapters/osf-adapter';
 
 moduleFor('adapter:osf-adapter', 'Unit | Adapter | osf adapter', {
-  // Specify the other units that are required for this test.
-  // needs: ['serializer:foo']
+    needs: [
+        'model:user', 'model:node', 'model:institution', 'model:registration', 'model:log', 'model:comment', 'model:contributor', 'model:file-provider', 'model:node-link', 'model:draft-registration',
+        'adapter:osf-adapter', 'adapter:node', 'adapter:user',
+        'serializer:node',
+        'service:session',
+        'transform:links', 'transform:embed'
+    ],
+    beforeEach() {
+        manualSetup(this.container);
+    }
+});
+
+test('#buildURL appends a trailing slash if missing', function(assert) {
```

Comment:
Small suggestion based on past regressions: can you please also add a test for

"Does not append a trailing slash if one is already present"